docker/elasticsearch/dataprocess-index-config.json

```diff
@@ -140,90 +139,92 @@
     }
   },
   "mappings": {
-    "properties": {
-      "browsePaths": {
-        "type": "text",
-        "fields": {
-          "length": {
-            "type": "token_count",
-            "analyzer": "slash_pattern"
-          }
+    "doc": {
+      "properties": {
+        "browsePaths": {
+          "type": "text",
+          "fields": {
+            "length": {
+              "type": "token_count",
+              "analyzer": "slash_pattern"
+            }
+          },
+          "analyzer": "custom_browse_slash",
+          "fielddata": true
         },
-        "analyzer": "custom_browse_slash",
-        "fielddata": true
-      },
-      "origin": {
-        "type": "keyword",
-        "fields": {
-          "ngram": {
-            "type": "text",
-            "analyzer": "custom_ngram"
-          }
+        "origin": {
+          "type": "keyword",
+          "fields": {
+            "ngram": {
+              "type": "text",
+              "analyzer": "custom_ngram"
+            }
+          },
+          "normalizer": "my_normalizer"
         },
-        "normalizer": "my_normalizer"
-      },
-      "hasOwners": {
-        "type": "boolean"
-      },
-      "name": {
-        "type": "keyword",
-        "fields": {
-          "data_process_pattern_ngram": {
-            "type": "text",
-            "analyzer": "data_process_pattern_ngram"
-          },
-          "delimited": {
-            "type": "text",
-            "analyzer": "delimit"
-          },
-          "ngram": {
-            "type": "text",
-            "analyzer": "custom_ngram"
-          },
-          "pattern": {
-            "type": "text",
-            "analyzer": "data_process_pattern"
-          }
+        "hasOwners": {
+          "type": "boolean"
         },
-        "normalizer": "my_normalizer"
-      },
-      "num_inputs": {
-        "type": "long"
-      },
-      "num_outputs": {
-        "type": "long"
-      },
-      "owners": {
-        "type": "text",
-        "fields": {
-          "ngram": {
-            "type": "text",
-            "analyzer": "comma_pattern_ngram"
-          }
+        "name": {
+          "type": "keyword",
+          "fields": {
+            "data_process_pattern_ngram": {
```

Comment:
Apologies for not reviewing this earlier. You seem to have created several subfields under `name` but I don't see them being used in either the search or autocomplete query template i.e. `dataProcessESSearchQueryTemplate.json` or `dataProcessESAutocompleteQueryTemplate.json`. Do you plan to add the support later?

Reply:
I think `dataProcessESSearchQueryTemplate.json` has it? 
```
"fields": [
          "name^4",
          "orchestrator"
         ],
        "default_operator": "and",
```

Reply:
I meant the subfields under `name` like `data_process_pattern_ngram`, `delimited`, etc. They don't seem to be used in search or autocomplete query template?